core/tx.go

```diff
@@ -29,7 +29,7 @@ type OperatorSetParam struct {
 
 type OperatorStakes map[QuorumID]map[OperatorIndex]OperatorStake
 
-type Transactor interface {
+type TransactorOld interface {
```

Comment:
not deleting this yet? I don't think it is used anywhere

Reply:
+1